e2e/app/page/admin-user-audit-page.ts

```diff
@@ -0,0 +1,16 @@
+import { Page } from 'puppeteer';
+import AuthenticatedPage from 'app/page/authenticated-page';
+import { waitForDocumentTitle, waitWhileLoading } from 'utils/waits-utils';
+
+const PageTitle = 'User Audit';
+
+export default class UserAuditPage extends AuthenticatedPage {
+  constructor(page: Page) {
+    super(page);
+  }
+
+  async isLoaded(): Promise<boolean> {
+    await Promise.all([waitForDocumentTitle(this.page, PageTitle), waitWhileLoading(this.page)]);
```

Comment:
`waitWhileLoading` call in `isLoaded` is not needed because it is already called in `waitForLoad`.